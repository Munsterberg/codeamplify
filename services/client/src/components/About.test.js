import React from 'react';
import { shallow } from 'enzyme';

import About from './About';

describe('<About />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<About />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.length).toBe(1);
  });

  it('should render h1 text', () => {
    const elem = wrapper.find('h1');
    expect(elem.length).toBe(1);
    expect(elem.text()).toBe('About Page');
  });
});