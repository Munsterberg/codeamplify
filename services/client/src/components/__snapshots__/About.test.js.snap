// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render correctly 1`] = `
<div>
  <h1>
    About Page
  </h1>
</div>
`;
